kitsune/kbadge/models.py

```diff
@@ -157,36 +85,16 @@ def __call__(self, instance, filename):
                                 ext=self.ext)
 
 
-class JSONField(models.TextField):
```

Comment:
I guess this was just already superfluous?

Reply:
Yep, this was something that could have been removed in the original `django-badger` port - I just missed it.